javascript-modal/styles.css

```diff
@@ -0,0 +1,68 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.open-button {
+  position: fixed;
```

Comment:
This doesn't need to be `fixed`

Reply:
The only problem is that if I delete that property then my page looks like this
![image](https://user-images.githubusercontent.com/76715055/106806111-63896c00-661c-11eb-892c-605e74b87150.png)

I don't think I wrote the cleanest HTML in the beginning so that later made me have to tinker with CSS in order to get the same output as you did in the gif.  My approach was definitely a messy one.

Reply:
@ErikModica If you have questions about how to achieve this, please let us know.